bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -537,7 +539,8 @@ spec:
         hostPath:
           path: /var/lib/ovn/data
       - name: run-openvswitch
-        emptyDir: {}
+        hostPath:
```

Comment:
@trozet can you add a `type: Directory` here so we never accidentally create this?